parity/cli/mod.rs

```diff
@@ -800,7 +800,7 @@ usage! {
 
 			ARG arg_logging: (Option<String>) = None, or |c: &Config| c.misc.as_ref()?.logging.clone(),
 			"-l, --logging=[LOGGING]",
-			"Specify the logging level. Must conform to the same format as RUST_LOG.",
+			"Specify the general  logging level (error, warn, info, debug or trace). It can also be set for a specific module. Example: '-l sync=debug, rpc=trace',
```

Comment:
Missing `"` in the end, and extra space after `general`. ;)

Reply:
fixed, that was obviously written too quickly :)